src/library/Uno.Themes.WinUI.Markup/Theme/TextBlock.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class TextBlock
+		{
+			public static class Styles
```

Comment:
I think there isn't supposed to be a toplevel `Styles`. It goes something like this

TextBlock -> Default -> Typography -> FontWeight/Family
                                  -> TextTrimming
                                  -> TextWrapping
                                  -> etc.
                -> Display -> Large     -> CharacterSpacing
                                                     -> etc.
                                  -> Medium -> ...
                                                     -> etc.
                                   -> etc.                                   
                -> Headline -> ...
                -> ...


@kazo0

Reply:
This is a special case, so `TextBlock.Styles.DisplayLarge/HeadlineLarge/etc.` makes sense, these are styles of the TextBlock control